src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,49 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int LENGTH_STORAGE = 10;
+    private K[] keys;
+    private V[] values;
+    private int size = 0;
```

Comment:
no need to set primitives to their default values
```suggestion
    private int size;
```